frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -93,6 +93,21 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 )
               }
             </Nav>
+            <Nav className="mr-auto">
+              {
+                hasRole(currentUser, "ROLE_USER") && (
+
+                  <NavDropdown title="Collegiate Subreddits" id="appnavbar-CollegiateSubreddits-dropdown" data-testid="appnavbar-CollegiateSubreddits-dropdown" >
+                    <NavDropdown.Item href="/collegiatesubreddits/list" data-testid="appnavbar-CollegiateSubreddits-list">List</NavDropdown.Item>
+                    {
+                      hasRole(currentUser, "ROLE_ADMIN") && (
+                        <NavDropdown.Item href="/collegiatesubreddits/create" data-testid="appnavbar-CollegiateSubreddits-create">Create</NavDropdown.Item>
```

Comment:
The routes should be named the same casing as the API routes
```suggestion
                    <NavDropdown.Item href="/collegiateSubreddits/list" data-testid="appnavbar-CollegiateSubreddits-list">List</NavDropdown.Item>
                    {
                      hasRole(currentUser, "ROLE_ADMIN") && (
                        <NavDropdown.Item href="/collegiateSubreddits/create" data-testid="appnavbar-CollegiateSubreddits-create">Create</NavDropdown.Item>
```